Setup instructions for external-tests in the README.md

It would be nice to include some setup instructions in README.md so people who are not familiar with the external-testsuite can get things setup to build (and run) the tests.  Something similar to https://github.com/dyninst/examples README.md

I did try using the following cmake config which works for Fedora35 for the examples:

`cmake . -DDyninst_DIR=path/to/Dyninst/install/lib/cmake/Dyninst`

With external-tests things fail; pretty quickly:

```
$ make
Consolidate compiler generated dependencies of target common_insertion_operators
[ 14%] Built target common_insertion_operators
Consolidate compiler generated dependencies of target df_insertion_operators
[ 28%] Built target df_insertion_operators
Consolidate compiler generated dependencies of target da_insertion_operators
[ 35%] Linking CXX executable insertion_operators
/usr/bin/ld: CMakeFiles/da_insertion_operators.dir/insertion_operators.cpp.o: in function `main':
insertion_operators.cpp:(.text+0x20): undefined reference to `operator<<(std::ostream&, BPatch_basicBlock&)'
collect2: error: ld returned 1 exit status
make[2]: *** [dyninstAPI/CMakeFiles/da_insertion_operators.dir/build.make:98: dyninstAPI/insertion_operators] Error 1
make[1]: *** [CMakeFiles/Makefile2:242: dyninstAPI/CMakeFiles/da_insertion_operators.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -1,3 +1,13 @@
-# External Tests
+# External Tests for [Dyninst](https://github.com/dyninst/dyninst)
 
-These are tests for Dyninst which do not fit within the framework of the test suite. Specifically, they are intended to test system-level libraries and functionality.
+These are tests for Dyninst which do not fit within the framework of the [test suite](https://github.com/dyninst/testsuite).
+
+## Building
+
+Before building any of these examples, you need an existing build of Dyninst (see the Dyninst [wiki](https://github.com/dyninst/dyninst/wiki/Building-Dyninst) for details).
+
+To configure the build, you can use
+
+    cmake . -DDyninst_DIR=path/to/Dyninst/install/lib/cmake/Dyninst
+    cmake --build .
+    ctest .

--- a/README.md
+++ b/README.md
@@ -10,4 +10,9 @@ To configure the build, you can use
 
     cmake . -DDyninst_DIR=path/to/Dyninst/install/lib/cmake/Dyninst
     cmake --build .
+
+## Running
+
+Most of the tests are designed to examine some aspect of compiling or linking Dyninst. There are, however, a few tests that have a runtime aspect. These tests can be run using CTest like so:
+
     ctest .

